lib/facter/mcollective_configured.rb

```diff
@@ -0,0 +1,5 @@
+Facter.add('mcollective_configured') do
```

Comment:
The only piece I really care about is whether it was configured. We'll always install mcollective as part of a puppet-agent install, and the puppet-agent install should take care of uninstalling mcollective if it existed before.